nominal/experimental/compute/_buckets.py

```diff
@@ -54,12 +95,17 @@ def _compute_buckets(
     start: api.Timestamp,
     end: api.Timestamp,
     buckets: int,
-) -> Iterable[tuple[api.Timestamp, scout_compute_api.NumericBucket]]:
+) -> Iterable[Bucket]:
```

Comment:
btw the fact that API functions return API objects (rather than client lib types) is intentional. I like to be able to move these functions around without needing to worry about import shenanigans, sometimes returning client lib types from API calls results in cyclic imports